emscripten/MarkerlessARJS_bindings.cpp

```diff
@@ -68,7 +68,7 @@ class ARPipeline_em {
 
   bool processFrame(const emscripten::val inputFrame, size_t width, size_t height) {
      auto u8 = emscripten::vecFromJSArray<uint8_t>(inputFrame);
-    cv::Mat inputFrameMat(height, width, CV_8UC1, u8.data());
+    cv::Mat inputFrameMat(height, width, CV_8UC4, u8.data());
```

Comment:
we are passing data from ImageData as Uint8ClampedArray as RGBA. S the `inputFramMat` should be load data accordingly.